_posts/2016-08-17-improve-teams-communication-with-user-stories.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Giulia Mantuano
+comments: true
+date: 2016-08-17 14:25:09+00:00
+layout: post
+asset-type: post
+slug: improve-teams-communication-with-user-stories
+title: Improve team's communication with User Stories
+image:
+    src: /assets/img/custom/blog/user-stories-sketchnote.jpg
+category: agile
+tags:
+- scrum
+- user stories
+- team
+---
+
+The challenge of Software development is about building a product which fulfills both business and users expectations. 
+
+The focus for each role within an Agile team is different: Project Managers want to see progress and want a quality product that is in time and value for money; Product Manager wants flexibility; Testers want to measure; Developers wants to write outstanding and sustainable code; Users are looking for a product which helps accomplish their tasks. How is it possible to find an agreement between all of these needs and build a product which could be a good compromise between building the right thing, building the thing right and building it in time?
+
+In order to build a shared understanding of the product vision, goals and values it is necessary to have every team member involved in the process (real users included!). User Stories are a tool that helps accomplish that: they improve communication within teams and stakeholders and reduce the risk of misinterpreting product development requirements.
+
+### What is a User Story?
+
+A User Story is a short sentence written from the user’s perspective which describes a valuable functionality our software or system will have.
+
+A User Story is made by 3 elements:
+
+ - Card
+ - Conversation
+ - Confirmation
+
+**The Card** is the written description of the feature to implement. It must be short, 1 or 2 sentences maximum, because it acts as a reminder to discuss the feature during the Conversation phase.
+
+**The Conversation** is the most important element of a User Story. To avoid misinterpretation the Customer Team (Product Manager, Project Manager, Tester, Interaction Designer, Users) and the Developers Team start a discussion around the description to identify its details and ensure that the developer will build the right solution meeting the needs of the user.
+
+**The Confirmation** is the criteria for when a story can be considered "DONE". The criteria may be written as acceptance tests.
+
+A User Story is always written from the perspective of the user. Common description formats that can help teams new to Agile practices as SCRUM are:
+
+> As a < user role > I < want, need, can, etc > < goal > so that < reason >
```

Comment:
Write these two formats in this fashion (easier to read, and also almost always written like this in practise):
```
As a < user role > 
I < want, need, can, etc > < goal > 
So that < reason >
```